packages/pwa-kit-react-sdk/scripts/prepare-dist.js

```diff
@@ -35,7 +35,7 @@ const catcher = (message) => (error) => {
  */
 const getPackageFiles = () => {
     let output = ''
-    const child = exec(`npm pack --dry-run --json --ignore-scripts`)
+    const child = exec(`npx npm-packlist`)
```

Comment:
curious why did you make this change?

Reply:
I see that you have explained this in the PR description

> When preparing the list of files to include in /dist, use `npx npm-packlist` instead of `npm pack --dry-run --json --ignore-scripts` as `npm pack --ignore-scripts` no longer skips pre and post scripts on npm 7.

I guess a sanity check, have you compared the pack list before and after to ensure we are still packaging the same files?

Reply:
Yes, I have done a compare of the packlist before and after and they are the same. (The change that adds CHANGELOG.md in package.json is required to make this happen. Not sure why, but without that line, packlist would ignore CHANGELOG.md)

Reply:
👍 awesome